packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.scss

```diff
@@ -0,0 +1,5 @@
+.ellipsis {
+  max-width: 100px;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
There might be a bootstrap 4 utility for that

Reply:
did not work well out of the box with table.